runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperConfig.java

```diff
@@ -1,264 +1,274 @@
+	)
+	default boolean swapAdmire()
+	{
+		return true;
+	}
+
+	@ConfigItem(
+		keyName = "swapAssignment",
+		name = "Assignment",
+		description = "Swap Talk-to with Assignment for Slayer Masters. This will take priority over swapping Trade."
+	)
+	default boolean swapAssignment()
+	{
+		return true;
+	}
+
+	@ConfigItem(
+		keyName = "swapBanker",
+		name = "Bank",
+		description = "Swap Talk-to with Bank on Bank NPC<br>Example: Banker"
+	)
+	default boolean swapBank()
+	{
+		return true;
+	}
+
+	@ConfigItem(
+		keyName = "swapBirdhouseEmpty",
+		name = "Birdhouse",
+		description = "Swap Interact with Empty for birdhouses on Fossil Island"
+	)
+	default boolean swapBirdhouseEmpty()
+	{
+		return true;
+	}
+
+	@ConfigItem(
+		keyName = "swapBones",
+		name = "Bury",
+		description = "Swap Bury with Use on Bones"
+	)
+	default boolean swapBones()
+	{
+		return false;
+	}
+
+	@ConfigItem(
+		keyName = "swapChase",
+		name = "Chase",
+		description = "Allows to left click your cat to chase"
+	)
+	default boolean swapChase()
+	{
+		return true;
+	}
+
+	@ConfigItem(
+		keyName = "claimSlime",
+		name = "Claim Slime",
+		description = "Swap Talk-to with Claim Slime from Morytania diaries"
+	)
+	default boolean claimSlime()
+	{
+		return true;
+	}
+
+	@ConfigItem(		
+		keyName = "swapDarkMage",
+		name = "Repairs",
+		description = "Swap Talk-to with Repairs for Dark Mage"
+	)
+	default boolean swapDarkMage()
+	{
+		return true;
+	}
+
+	@ConfigItem(
+		keyName = "swapDecant",
+		name = "Decant",
+		description = "Swap Talk-to with Decant for Bob Barter and Murky Matt at the Grand Exchange."
+	)
+	default boolean swapDecant()
+	{
+		return false;
+	}
+
+	@ConfigItem(
+		keyName = "swapExchange",
+		name = "Exchange",
+		description = "Swap Talk-to with Exchange on NPC<br>Example: Grand Exchange Clerk, Tool Leprechaun, Void Knight"
+	)
+	default boolean swapExchange()
+	{
+		return true;
+	}
+
+	@ConfigItem(
+		keyName = "swapFairyRing",
+		name = "Fairy ring",
+		description = "Swap Zanaris with Last-destination or Configure on Fairy rings"
+	)
+	default FairyRingMode swapFairyRing()
+	{
+		return FairyRingMode.LAST_DESTINATION;
+	}
+
+	@ConfigItem(
+		keyName = "swapHarpoon",
+		name = "Harpoon",
+		description = "Swap Cage, Big Net with Harpoon on Fishing spot"
+	)
+	default boolean swapHarpoon()
+	{
+		return false;
+	}
+
+	@ConfigItem(
+		keyName = "swapHomePortal",
+		name = "Home",
+		description = "Swap Enter with Home or Build or Friend's house on Portal"
+	)
+	default HouseMode swapHomePortal()
+	{
+		return HouseMode.HOME;
+	}
+
+	@ConfigItem(
+		keyName = "swapPickpocket",
+		name = "Pickpocket on H.A.M.",
+		description = "Swap Talk-to with Pickpocket on H.A.M members"
+	)
+	default boolean swapPickpocket()
+	{
+		return true;
+	}
+
+	@ConfigItem(
+		keyName = "swapHelp",
+		name = "Help Arceuus Librarians",
+		description = "Swap Talk-to with Help on Arceuus Librarians"
```

Comment:
Not that it really matters.

```suggestion
		description = "Swap Talk-to with Help on Kourend Library customers"
```